react/ProductWrapper.js

```diff
@@ -53,12 +59,17 @@ const ProductWrapper = ({
     [loading, hasProductData]
   )
 
+  const SSRLoading =
```

Comment:
:thonking: a bit strange but ok

Reply:
(this being a component or null instead of a boolean)

Reply:
I will change to make it more clear.